bootstrap.sh

```diff
@@ -4,9 +4,13 @@ set -eu
 ## Make sure we have Ruby 3 installed directly on the instance
 amazon-linux-extras install -y ruby3.0
 
-## Install Terraform
-curl "https://releases.hashicorp.com/terraform/1.2.7/terraform_1.2.7_linux_amd64.zip" -o "terraform.zip"
-sudo unzip ./terraform.zip -d /usr/local/bin
+# Install Kubectl and Helm CLIs
+wget -q https://s3.us-west-2.amazonaws.com/amazon-eks/1.23.17/2023-05-11/bin/linux/amd64/kubectl -O /usr/local/bin/kubectl
```

Comment:
we should fix the underlying issue causing kubectl > 1.23.x to not work rather than pin to 1.23.x. kubernetes only supports a version skew of 1 minor version, so pinning 1.23.x means that as soon as we need to upgrade our clusters again (which we will before the end of the year) we will be out of the supported version skew.  Since this seems to affect all versions > 1.23.x we will have to resolve the issue to continue upgrading.